sphinxcontrib/spelling/builder.py

```diff
@@ -9,14 +9,21 @@
 import importlib
 import io
 import os
+import os.path
```

Comment:
given the previous line, this seems redundant
```suggestion
```

Reply:
Done